sdk/automation/Azure.ResourceManager.Automation/src/Generated/Models/AutomationModuleField.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.Automation.Models
+{
+    /// <summary>
+    /// Information about a field of a type.
+    /// Serialized Name: TypeField
+    /// </summary>
+    public partial class AutomationModuleField
+    {
+        /// <summary> Initializes a new instance of AutomationModuleField. </summary>
+        internal AutomationModuleField()
+        {
+        }
+
+        /// <summary> Initializes a new instance of AutomationModuleField. </summary>
+        /// <param name="name">
+        /// Gets or sets the name of the field.
+        /// Serialized Name: TypeField.name
+        /// </param>
+        /// <param name="typeFieldType">
+        /// Gets or sets the type of the field.
+        /// Serialized Name: TypeField.type
+        /// </param>
+        internal AutomationModuleField(string name, string typeFieldType)
+        {
+            Name = name;
+            TypeFieldType = typeFieldType;
+        }
+
+        /// <summary>
+        /// Gets or sets the name of the field.
+        /// Serialized Name: TypeField.name
+        /// </summary>
+        public string Name { get; }
+        /// <summary>
+        /// Gets or sets the type of the field.
+        /// Serialized Name: TypeField.type
+        /// </summary>
+        public string TypeFieldType { get; }
```

Comment:
Bad name, suggest renaming to `FieldType`

Reply:
fixed